test/test_exporter.py

```diff
@@ -327,127 +343,128 @@ def test_entire_batch_fail(self, mock_const, mock_post):
         mock_post.assert_called_once_with(
             self._ingest_endpoint,
-            data="my.instr,l1=v1,l2=v2,dt.metrics.source=opentelemetry gauge,10 " + str(self._test_timestamp) + "\n",
+            data="my.instr,l1=v1,l2=v2,dt.metrics.source=opentelemetry gauge,10 {0}\n".format(
+                str(int(self._test_timestamp_nanos / 1000000))),
             headers=self._headers)
 
     @patch.object(requests.Session, 'post')
-    def test_min_max_sum_count_aggregator(self, mock_post):
-        mock_post.return_value = self._get_session_response()
-
-        aggregator = aggregate.MinMaxSumCountAggregator()
-        aggregator.update(100)
-        aggregator.update(1)
-        self._update_value(aggregator, 10)
-        aggregator.last_update_timestamp = self._test_timestamp
-        record = self._create_record(aggregator)
+    def test_histogram_reported_as_gauge(self, mock_post):
+        data_point = Histogram(bucket_counts=[1, 2, 4, 5],
+                               explicit_bounds=[0, 5, 10],
+                               sum=36,
+                               aggregation_temporality=AggregationTemporality.DELTA,
+                               time_unix_nano=self._test_timestamp_nanos,
+                               start_time_unix_nano=self._test_timestamp_nanos)
+        metric = self._create_record(data_point)
 
         exporter = DynatraceMetricsExporter()
-        result = exporter.export([record])
+        result = exporter.export([metric])
 
-        self.assertEqual(MetricsExportResult.SUCCESS, result)
+        self.assertEqual(MetricExportResult.SUCCESS, result)
         mock_post.assert_called_once_with(
             self._ingest_endpoint,
-            data="my.instr,l1=v1,l2=v2,dt.metrics.source=opentelemetry gauge,min=1,max=100,sum=111,count=3 " + str(self._test_timestamp) + "\n",
+            data="my.instr,l1=v1,l2=v2,dt.metrics.source=opentelemetry gauge,min=3,max=3,sum=36,count=12 {0}\n"
+                .format(str(int(self._test_timestamp_nanos / 1000000))),
             headers=self._headers)
 
     @patch.object(requests.Session, 'post')
-    def test_last_value_aggregator(self, mock_post):
+    def test_multiple_records(self, mock_post):
         mock_post.return_value = self._get_session_response()
 
-        aggregator = aggregate.LastValueAggregator()
-        self._update_value(aggregator, 20)
-
-        record = self._create_record(aggregator)
-
+        records = []
+        records.append(self._create_record(self._create_sum_data_point(10)))
+        records.append(self._create_record(Gauge(time_unix_nano=self._test_timestamp_nanos, value=20)))
+        records.append(self._create_record(Histogram(bucket_counts=[1, 2, 4, 5],
+                                                     explicit_bounds=[0, 5, 10],
+                                                     sum=36,
+                                                     aggregation_temporality=AggregationTemporality.DELTA,
```

Comment:
Is this explicitly needed here?

Reply:
Yes, `aggregation_temporality` is required for `Histogram` and `Sum`.